src/Codenizer.HttpClient.Testable/IRequestBuilder.cs

```diff
@@ -87,6 +88,13 @@ public interface IRequestBuilder
         /// <param name="contentType">A MIME content type (for example text/plain)</param>
         /// <returns>The current <see cref="IRequestBuilder"/> instance</returns>
         IRequestBuilder AndContentType(string contentType);
+
+        /// <summary>
+        /// Respond to a request that matches the authorization header
+        /// </summary>
+        /// <param name="authenticationHeader">A authorization header value</param>
+        /// <returns>The current <see cref="IRequestBuilder"/> instance</returns>
+        IRequestBuilder Authorization(AuthenticationHeaderValue authenticationHeader);
```

Comment:
Could you change the name to `AndAuthorization` so that the usage is similar to `AndContentType()`?

So instead of

```csharp
handler
    .RespondTo()
    .Post()
    .ForUrl("/api/infos/")
    .Authorization(new AuthenticationHeaderValue())
    .With(HttpStatus.OK);
```

have

```csharp

handler
    .RespondTo()
    .Post()
    .ForUrl("/api/infos/")
    .AndAuthorization(new AuthenticationHeaderValue())
    .With(HttpStatus.OK);
```